webapp/sources/rudder/rudder-web/src/main/webapp/secure/configurationManager/techniqueEditor.html

```diff
@@ -14,6 +14,7 @@
       </div>
     </div>
     <div id="technique-editor">
+    </div>
```

Comment:
is this related to the local storage or just a side enhancement ? (I don't see how it could be related, but sometime I just don't get js, so...)

Reply:
this make the js script disappear, as it is replaced by the initialisation of the elm app. This make it impossible to track if a changes was made in the js scripts. This is a small side change yes